docs/_static/css/github_issue_links.css

```diff
@@ -0,0 +1,6 @@
+.github-issue-link-container {
+    padding-right: 0.5rem;
+}
+.github-issue-link {
```

Comment:
As a suggestion, I think it calls a little more attention if we make it
`font-weight: bold`
Feel free to ignore if disagree though